### PR TITLE
update Tuya doc

### PR DESCRIPTION
**Pull request by zlinoliver**
## Proposed change

- Update "image_002.png" image.
- Fix 404 Github wiki link issue.
- Update content related to 1106 error code.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by zlinoliver** [comment]
Hi @frenck @balloob , please help to review this PR, thanks!

File: source/_integrations/tuya.markdown [hunk 1]
```diff
@@ -97,18 +98,18 @@ Click the created project to enter the `Project Overview` page and get the `Auth
 {% configuration_basic %}
 
 "1004: sign invalid":
-  description: Incorrect Access ID or Access Secret. Please refer to [Tuya credential table](https://github.com/tuya/tuya-home-assistant/wiki/Install-Tuya-v2?_source=d10de34623e3daca5b02e3c31528a0c4#3-enter-your-tuya-credential).
+  description: Incorrect Access ID or Access Secret. Please refer to [Install Tuya Integration](https://github.com/tuya/tuya-home-assistant/blob/master/docs/install.md).
```
Comment (frenck) [inline source/_integrations/tuya.markdown]: This is the Tuya install guide. Why link this to the same content outside of our website?

File: source/_integrations/tuya.markdown [hunk 2]
```diff
@@ -97,18 +98,18 @@ Click the created project to enter the `Project Overview` page and get the `Auth
...
 
     - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields. Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
 
-    - Incorrect availability zone: See [Availability Zone](https://github.com/tuya/tuya-home-assistant/wiki/Tuya-IoT-Platform-Configuration-Guide-Using-Smart-Home-PaaS#region--available-zone-correspondence) and select the correct availability zone.
+    - Incorrect country. You must select the region of your account of the Tuya Smart app or Smart Life app.
 
 "1100: param is empty":
-  description: Empty parameter of username or app. Fill the parameters refer to [Tuya credential table](https://github.com/tuya/tuya-home-assistant/wiki/Install-Tuya-v2?_source=d10de34623e3daca5b02e3c31528a0c4#3-enter-your-tuya-credential).
+  description: Empty parameter of username or app. Fill the parameters refer to [Install Tuya Integration](https://github.com/tuya/tuya-home-assistant/blob/master/docs/install.md).
```
Comment (frenck) [inline source/_integrations/tuya.markdown]: Same comment as above.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tuya.markdown (hunk 1 above)
Comment (zlinoliver) [reply source/_integrations/tuya.markdown]: You are right, remove the link and refer to the configuration part in the website.

File: source/_integrations/tuya.markdown (hunk 2 above)
Comment (zlinoliver) [reply source/_integrations/tuya.markdown]: Remove the link and refer to the configuration part in the website.

**Review by frenck — APPROVED** [review]
Thanks, @zlinoliver 👍

**Pull request merged**